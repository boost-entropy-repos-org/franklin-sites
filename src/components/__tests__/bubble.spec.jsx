import React from 'react';
import renderer from 'react-test-renderer';
import Bubble from '../bubble';

describe('Long number component', () => {
  test('should render with defaults', () => {
    const component = renderer.create(<Bubble value={10} />).toJSON();
    expect(component).toMatchSnapshot();
  });
  test('should render bigger than 100', () => {
    const component = renderer.create(<Bubble value={101} />).toJSON();
    expect(component).toMatchSnapshot();
  });
  test('should render different size and colour', () => {
    const component = renderer
      .create(<Bubble value={101} colourClass="colour-light-grey" size="large" />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
