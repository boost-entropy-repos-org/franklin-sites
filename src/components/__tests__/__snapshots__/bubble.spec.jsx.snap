// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Long number component should render bigger than 100 1`] = `
<span
  className="bubble--medium colour-sea-blue"
>
  99+
</span>
`;

exports[`Long number component should render different size and colour 1`] = `
<span
  className="bubble--large colour-light-grey"
>
  99+
</span>
`;

exports[`Long number component should render with defaults 1`] = `
<span
  className="bubble--medium colour-sea-blue"
>
  10
</span>
`;
